examples/generalPermissionManager.ts

```diff
@@ -21,16 +21,13 @@ window.addEventListener('load', async () => {
 
   // Instantiate the API
   const polymathAPI = new PolymathAPI(params);
-  // Get some poly tokens in your account and the security token
-  const web3Wrapper = new Web3Wrapper(params.provider, {
-    gasPrice: params.defaultGasPrice,
-  });
-  const address = await web3Wrapper.getAvailableAddressesAsync();
 
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  // Get some poly tokens in your account and the security token
+  const myAddress = await polymathAPI.getAccount();
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: myAddress });
+  await polymathAPI.polyToken.transfer({
```

Comment:
Same as above. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.